lib/processor.js

```diff
@@ -170,23 +166,26 @@ function parse(text) {
 
         // Line content
         } else {
-            next = match(/[^\r\n]*/);
+            next = match(/^[^\r\n]*/);
```

Comment:
Adding the anchor makes no difference to the logic of the program. It was already only reading to the end of the line (until it saw `\r` or `\n`). Combined with the earlier check for `/^/r?/n/`, this will only be hit when the line does not begin with a newline. That also means it will always match. Adding the anchor would be an optimization if it ever didn't match, but now it shouldn't change anything. It does, however, make it consistent with the other (non-error) regexes, which have anchors.